package com.example.dto;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

public class LoginRequest {

	@NotBlank(message = "Username is required") 
	@Size(min = 4, max = 30, message = "Username must be between 4 and 30 characters")
	private String username;
	
	@NotBlank(message = "Password is required")
	@Size(min = 8, max = 50, message = "Password must be between 8 and 50 characters")
	private String password;

	    public String getUsername() {
	        return username;
	    }

	    public void setUsername(String username) {
	        this.username = username;
	    }

	    public String getPassword() {
	        return password;
	    }

	    public void setPassword(String password) {
	        this.password = password;
	    }
	}

